core/platform/email/gae_email_services.py

```diff
@@ -22,6 +22,15 @@
 
 
 def get_incoming_email_address(reply_to_id):
+    """Returns string detailing the id of the reply-client.
```

Comment:
This is not clear are you sure it returns Id? and I'm not getting what is "reply-client" here.

Reply:
You can write this instead `Converts a reply-to ID to an email address that is used for identifying email replies.`.